composer.json

```diff
@@ -21,7 +21,7 @@
         "php": ">=7.2",
         "cakephp/cakephp": "^4.0",
         "cakephp/plugin-installer": "^1.0",
-        "wyrihaximus/twig-view": "^5.0.1"
+        "cakephp/twig-view": "^1.0.0"
```

Comment:
```suggestion
        "cakephp/twig-view": "^1.0"
```

Reply:
Why? 3 digit is the default, and same meaning :)